packages/next-ui/Snackbar/MessageSnackbarImpl.tsx

```diff
@@ -14,11 +14,12 @@ import React, { useEffect, useState } from 'react'
 import { IconSvg } from '../IconSvg'
 import { extendableComponent, breakpointVal } from '../Styles'
 import { iconClose, iconCheckmark, iconSadFace } from '../icons'
+import iconInfo from '../icons/info.svg'
 
 type Size = 'normal' | 'wide'
 type Variant = 'contained' | 'pill'
 
-export type MessageSnackbarProps = Omit<
+export type MessageSnackbarImplProps = Omit<
```

Comment:
The idea was that this is just an implementation detail, so the user didn't need to know